CHANGELOG.md

```diff
@@ -25,6 +25,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.25.6] - 2023-10-16
```

Comment:
Since this is just going into a feature branch, I would suggest that we just change this line to say "Unreleased" until we merge the feature/CRAYSAT-1740 branch into main.

```suggestion
## [Unreleased]
```

If you try to put a version number here already, it's just going to get out-of-date with the latest version in main anyway, and we might mistakenly reuse and existing version.